src/events.ts

```diff
@@ -1,5 +1,7 @@
+import { MessageBus } from "./ipc";
+
 export class EventArgs {
-    public readonly sender: any;
+    public readonly sender?: any;
```

Comment:
No sender for static events.  I can make it the class definition itself but I don't think that is useful.